tests/rt.conf

```diff
@@ -31,6 +31,10 @@ RUN | cpld_control_gfsv17_iau                           | - noaacloud
 RUN | cpld_restart_gfsv17                               | - noaacloud                           |          | cpld_control_gfsv17
 RUN | cpld_mpi_gfsv17                                   | - noaacloud                           |          |
 
+#SFS
+COMPILE | s2swa_32bit_pdlib_sfs  | intel | -DAPP=S2SWA -D32BIT=ON -DHYDRO=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | - noaacloud | fv3 |
+RUN | cpld_gfsv17_sfs                                   | - noaacloud                           | baseline |
```

Comment:
@XiaqiongZhou-NOAA Mixing gfsv17 and sfs names/configurations is confusing to me. Are you planning to add more "sfs" tests? Renaming (e.g., s2swa_32bit_pdlib_hydro and cpld_hydro_gfsv17) seems more clear to me wrt other RT tests.

Reply:
@NickSzapiro-NOAA : This is for the new application SFS. I will keep "sfs" since this is for the new application SFS and change it to "cpld_sfs_hydro". Any comments?